gnc/pmc/CMakeLists.txt

```diff
@@ -15,30 +15,26 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.5)
 project(pmc)
 
-## Compile as C++14, supported in ROS Kinetic and newer
-add_compile_options(-std=c++14)
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+set( CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -Wall -O3 -fPIC" )
```

Comment:
didn't we used to have some extra flags, like WError?

Reply:
I copied this from a different package, I see that the examples have -Wall -Wextra -Wpedantic, I think I'll do a separate PR with all the packages updated. (The -fPIC I added because it's needed to export a library)